README.markdown

```diff
@@ -5581,12 +5581,21 @@ This method was introduced in the `0.5.0rc30` release.
 ngx.escape_uri
 --------------
 
-**syntax:** *newstr = ngx.escape_uri(str)*
+**syntax:** *newstr = ngx.escape_uri(str, type?)*
 
 **context:** *init_by_lua&#42;, init_worker_by_lua&#42;, set_by_lua&#42;, rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, header_filter_by_lua&#42;, body_filter_by_lua&#42;, log_by_lua&#42;, ngx.timer.&#42;, balancer_by_lua&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;, ssl_session_store_by_lua&#42;*
 
 Escape `str` as a URI component.
 
+Since `v0.10.16rc6`, this function accepts an optional `type` argument.
+```lua
+
+   type = 0 (Escape <code>str</code> as a URI)
+   type = 2 (Escape <code>str</code> as a URI component)
```

Comment:
You cannot use `<code>` inside a code block. It does not make sense. Maybe you should use markdown's bullet list here instead.